Makefile

```diff
@@ -6,15 +6,17 @@ all:
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	mkdir -p $(DESTDIR)/etc/neofetch
+	mkdir -p $(DESTDIR)$(PREFIX)/etc/neofetch
```

Comment:
You're right, I totally misssed that.

How are we going to un-hard-code it then? Right now we assume that `/etc/` exists and that it's writable. I thought this was ok, but according to #718 it's an issue. 

Hm